provision/acc_provision/acc_provision.py

```diff
@@ -238,6 +238,9 @@ def config_default():
             "hostagent_log_level": "info",
             "opflexagent_log_level": "info",
         },
+        "drop_log_config": {
+            "enable": True
+        },
```

Comment:
This is initializing part. You will have to read the configured value - look at config_adjust()
Ex: https://github.com/noironetworks/acc-provision/blob/master/provision/acc_provision/acc_provision.py#L504

Reply:
I don't have additional config, based on the knob, unlike istio, so I don't need to do anything more with config_adjust. I added the test case like you mentioned. Thanks for reviewing!